### DISCUSSION TITLE
seapp?

### DISCUSSION THREAD

**Discussion by korpe1** [SELinuxProject/selinux#425] [Q&A] [answered]
What seapp_contexts are and are them Android spesific?

**Comment by cgzones** [comment, marked as answer]
See https://github.com/SELinuxProject/selinux-notebook/blob/main/src/seandroid.md#seapp_contexts

**Reply by korpe1** [reply]
Are these only in format of policy source? I.e. generated policy is the same than on ordinary SELinux of Fedora Linux etc.?

**Reply by cgzones** [reply]
This is Android only. I am not familiar with the details though.

**Reply by korpe1** [reply]
Okay, thanks for you help :) I will read some docs and sources and hopefully I figure this out.
